tests/test_gpg.py

```diff
@@ -31,15 +31,24 @@ def get_local_fp():
             stderr=subprocess.DEVNULL,
         )
         results = subprocess.check_output(["gpg", "-k", "--with-fingerprint"], env=gpg_env)
-        return find_fp_from_gpg_output(results)
+        fingerprints = find_fps_from_gpg_output(results)
+
+        # Because we imported this key into a temporary directory,
+        # we should only have one key in the keyring.
+        if len(fingerprints) == 1:
```

Comment:
Is there a reason you didn't make this an assertion? Otherwise it seems like there needs to be an else case here that throws an exception or pytest.fails